### ISSUE TITLE
TS141.1: System did not prompt/notify that the shipment was skipped. 

### ISSUE THREAD

**Issue by agiebanez** [vasavah/vsv-fli-grn#145]
**Describe the bug**
System did not prompt/notify that the shipment was skipped. 

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by HannahCastro** [comment]
Apologies. The issue is that there is no prompt or notification when a shipment was skipped.

Example: There are 3 stops in the chain template: A - B - C. B has already received and released the shipment but didn't process the transaction in the system. Shipment has arrived at C's location so C will receive in the system. C was able to click on the "Receive" button but nothing is happening since B hasn't received it in the system.

**Expected behavior**
When C clicks on the "Receive" button, system should prompt C that he/she can't receive the shipment yet. System would also notify B that they have to receive the shipment in the system so that C can receive after.
When this happens, system should tag this as Late.

**Comment by zetaleona** [comment]
Hi!

We just want to ask if we will display the "Receive" and "Release" buttons for all users who will be part of the chain template. This is because of the current process, the "Receive" button only shows the next user in the chain template when the previous user clicks the "Release" button.

**Comment by zetaleona** [comment]
Hi!

We just want to ask if we will display the "Receive" and "Release" buttons for all users who will be part of the chain template? 
For the current process, the "Receive" button only shows the next user in the chain template when the previous user clicks the "Release" button.

**Comment by HannahCastro** [comment]
> Hi!
> 
> We just want to ask if we will display the "Receive" and "Release" buttons for all users who will be part of the chain template?
> For the current process, the "Receive" button only shows the next user in the chain template when the previous user clicks the "Release" button.

No, because this will be confusing to the users. We were previously suggesting if Logistics or Admin can have an override on the receiving for situations like this.

**Comment by VCI-Jojo** [comment]
@HannahCastro This ticket is related to #195 as per discussion with CTI and will be part of V1.1

For reference: Chain Template: add override if there will be points on the chain template that will be skipped
